app/services/event.service.js

```diff
@@ -213,27 +208,153 @@
       } else { //Reset page and empty the list
         service.page = 1;
         service.eventsList = [];
+        service.reserve = [];
       }
 
       //Add pagination query
       filterString = filterString + '&page=' + service.page;
 
-      /** DEBUG **/ window.console.log(filterString);
+      /** DEBUG **/ //window.console.log(filterString);
```

Comment:
We leave the console log in to allow admins an easy way to check the filter string. Please leave this in.